Sources/App/Setup/Config+Setup.swift

```diff
@@ -6,7 +6,7 @@ extension Config {
     // allow fuzzy conversions for these types
     // (add your own types here)
     Node.fuzzy = [Row.self, JSON.self, Node.self]
-    
+    self.addConfigurable(middleware: AppMiddleware.init, name: "app-middleware")
```

Comment:
There too.

Reply:
Fixed (renamed `AppMiddleware` to `ClientMiddleware` everywhere)